curriculum/challenges/russian/08-coding-interview-prep/algorithms/implement-bubble-sort.russian.md

```diff
@@ -3,28 +3,30 @@ id: 8d5123c8c441eddfaeb5bdef
 title: Implement Bubble Sort
 challengeType: 1
 videoUrl: ''
-localeTitle: Реализация Bubble Sort
+localeTitle: Реализация Сортировки пузырьком
 ---
 
-## Description
-undefined
+## Описание
+Это первая из нескольких задач алгоритмов сортировки. Учитывая массив несортированных элементов, мы хотим иметь возможность возвращать отсортированный массив. Мы рассмотрим несколько различных методов для этого и изучим некоторые компромиссы между этими различными подходами. Хотя большинство современных языков имеют встроенные методы сортировки для таких операций, все же важно понять некоторые из общих базовых подходов и узнать, как они могут быть реализованы. Здесь мы увидим вид пузыря. Метод сортировки пузырьков начинается с начала несортированного массива и «выравнивает» несортированные значения до конца, итерации по массиву до тех пор, пока он не будет полностью отсортирован. Он делает это, сравнивая смежные элементы и заменяя их, если они не соответствуют порядку. Метод продолжает цикл через массив до тех пор, пока не произойдет своп, после чего массив будет отсортирован. Этот метод требует нескольких итераций через массив, а для средних и худших случаев имеет квадратичную временную сложность. Хотя это просто, в большинстве случаев это обычно нецелесообразно. Инструкции: Напишите функцию bubbleSort, которая принимает массив целых чисел в качестве входных данных и возвращает массив этих целых чисел в отсортированном порядке от наименьшего к наибольшему. Замечания:
+Мы вызываем эту функцию из-за кулис; тестовый массив, который мы используем, закомментирован в редакторе. Попробуйте logging array, чтобы увидеть ваш алгоритм сортировки в действии!
+
 
 ## Instructions
 undefined
```

Comment:
```
<section id="instructions">

</section>
```

Reply:
@ValeraS I have not tested this change after it was implemented.  How would we even test that this would fail right now?

Reply:
@RandellDawson locally you can run tests for any language:
```
cd curriculum && TEST_CHALLENGES_FOR_LANGS=russian npm test -- -f 'name of the challenge'
```